lib/Gitlab/Api/Users.php

```diff
@@ -448,4 +448,44 @@ public function removeImpersonationToken($user_id, $impersonation_token_id)
     {
         return $this->delete('users/'.$this->encodePath($user_id).'/impersonation_tokens/'.$this->encodePath($impersonation_token_id));
     }
+
+    /**
+     * @param int|string $user_id
```

Comment:
when is the user id a string?

Reply:
Thought maybe if someone saved it in his own database, but now I have a closer look at User.php I don't see it elsewhere, I'll change it!